dace/memlet.py

```diff
@@ -372,6 +364,10 @@ def try_initialize(self, sdfg: 'dace.sdfg.SDFG',
                 is_data_src = False
         self._is_data_src = is_data_src
 
+        # If subset is None, fill in with entire array
+        if (self.data is not None and self.subset is None):
+            self.subset = subsets.Range.from_array(sdfg.arrays[self.data])
```

Comment:
What happens for Streams? Does it insert "0"?

Reply:
yes :) this relates to your issue from before.